apps/docs/src/examples/extras/mesh-refraction-material/App.svelte

```diff
@@ -0,0 +1,10 @@
+<script lang="ts">
+  import { Canvas } from '@threlte/core'
+  import Scene from './Scene.svelte'
+</script>
+
+<div class="relative h-full w-full ">
+  <Canvas>
+    <Scene />
+  </Canvas>
+</div>
```

Comment:
Sorry, I missed this on my first review

```suggestion
<div>
  <Canvas>
    <Scene />
  </Canvas>
</div>
<style>
  div {
    height: 100%;
  }
</style>
```